website/static/js/share/results.js

```diff
@@ -94,7 +95,6 @@ var TitleBar = {
     }
 };
 
-/* Render the description of a single result. Will highlight the matched text */
```

Comment:
Let's bring back this comment, which was here before this PR. Also, I liked the comments that were removed in f79a90d4dd601c6e1bfefd0a0895cfdbb5b8335b. Let's bring those back, too! :-)